src/main/java/com/openshift/internal/restclient/model/volume/PersistentVolume.java

```diff
@@ -49,22 +49,22 @@ public PersistentVolume(ModelNode node, IClient client, Map<String, String[]> pr
 
     /**
      * @param unit
-     *            the designated unit. One of "Ki", "Mi", "Gi", "Ti", "Pi", "Ei".
+     *            the designated unit. One of "B", "K", "Ki", "M", "Mi", "G", "Gi",
+     *            "T", "Ti", "P", "Pi", "E", "Ei".
      * @return 0, if conversion not possible. Otherwise the capacity in given units.
      * @see {@link PersistentVolume#convert(String, MemoryUnit)}
      * @throws IllegalArgumentException
      *             if not supported {@code unit} given.
      */
     @Override
     public long getCapacity(String unit) {
-        String capacity = asString(PV_CAPACITY);
-        return convert(capacity, MemoryUnit.valueOf(unit));
+        return getCapacity(MemoryUnit.valueOf(unit));
```

Comment:
@adietish We've made a concerted effort to stay away from enums to make this client lenient.  Seems like here is a ase that's acceptible.